src/Mutator/Sort/Spaceship.php

```diff
@@ -74,6 +74,65 @@ public function mutate(Node $node): iterable
 
     public function canMutate(Node $node): bool
     {
-        return $node instanceof Node\Expr\BinaryOp\Spaceship;
+        if (!$node instanceof Node\Expr\BinaryOp\Spaceship) {
+            return false;
+        }
+
+        if ($this->isCompareWithZero($node)) {
+            return false;
+        }
+
+        return true;
+    }
+
+    private function isCompareWithZero(Node\Expr\BinaryOp\Spaceship $node): bool
+    {
+        $parentAttribute = $node->getAttribute('parent');
+
+        if ($parentAttribute instanceof Node\Expr\BinaryOp\Identical) {
+            if ($parentAttribute->right instanceof Node\Scalar\LNumber && $parentAttribute->right->value === 0) {
+                return true;
+            }
+
+            if ($parentAttribute->left instanceof Node\Scalar\LNumber && $parentAttribute->left->value === 0) {
+                return true;
+            }
+        }
+
+        if ($parentAttribute instanceof Node\Expr\BinaryOp\Equal) {
+            if ($parentAttribute->right instanceof Node\Scalar\LNumber && $parentAttribute->right->value === 0) {
+                return true;
+            }
+
+            if ($parentAttribute->right instanceof Node\Scalar\DNumber && $parentAttribute->right->value === 0.0) {
+                return true;
+            }
+
+            if (
+                $parentAttribute->right instanceof Node\Scalar\String_
+                && is_numeric($parentAttribute->right->value)
+                && ($parentAttribute->right->value === '0' || $parentAttribute->right->value === '0.0')
+            ) {
+                return true;
+            }
+
+            if ($parentAttribute->left instanceof Node\Scalar\LNumber && $parentAttribute->left->value === 0) {
+                return true;
+            }
+
+            if ($parentAttribute->left instanceof Node\Scalar\DNumber && $parentAttribute->left->value === 0.0) {
+                return true;
+            }
+
+            if (
+                $parentAttribute->left instanceof Node\Scalar\String_
+                && is_numeric($parentAttribute->left->value)
+                && ($parentAttribute->left->value === '0' || $parentAttribute->left->value === '0.0')
```

Comment:
Do you this `0 === (int) $parentAttribute->left->value` won't work?

Reply:
what if somebody decided to compare with 0.1 ? :)